### PR TITLE
[feature] Add support score calibration

### PR DESCRIPTION
**Pull request by Hunterhuan**
We support score calibration for Voxceleb recipe, and the results are listed as following:

ResNet34
image

ResNet34 LM
image

ResNet293 LM
image

ECAPA_TDNN1024 LM
image

We support score calibration for Cnceleb recipe, and the results are listed as following:
ResNet34
image

### CODE REVIEW

**Comment by czy97** [comment]
Score calibration reference:
- https://arxiv.org/pdf/2010.11255
- https://arxiv.org/pdf/2211.00815
hello, @Hunterhuan can you also add the hyperlink in the README file for score calibration reference.

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ pre-commit install  # for clean and tidy code
 ```
 
 ## 🔥 News
+* 2024.05.15: Add support for score calibration, see [#320](https://github.com/wenet-e2e/wespeaker/pull/320)
```
Comment (czy97) [inline README.md:63]: add hyper-links for score calibration:
https://arxiv.org/pdf/2010.11255 or our: https://arxiv.org/pdf/2211.00815

File: examples/cnceleb/v2/README.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 * Setup: fbank80, num_frms200, epoch150, ArcMargin, aug_prob0.6, speed_perturb (no spec_aug)
 * test_trials: CNC-Eval-Avg.lst
+
+* 🔥 UPDATE 2024.05.16: We update to support score calibration for cnceleb. It will improve the EER but degrade minDCF comparing with asnorm results.
+| Model                             | Params    | FLOPs   | LM  | AS-Norm   | Score Calibration | EER (%)   | minDCF (p=0.01)  |
```
Comment (czy97) [inline examples/cnceleb/v2/README.md:7]: It seems the markdown has the wrong format.

**Review by czy97 — COMMENTED** [review]
I have some comments on the README.md and examples/cnceleb/v2/README.md. @Hunterhuan  Can you check it?

**Comment by Hunterhuan** [comment]
> I have some comments on the README.md and examples/cnceleb/v2/README.md. @Hunterhuan Can you check it?

I have resolved these typo.

**Review by czy97 — APPROVED** [review]
Well Done!

**Pull request merged**